src/core/markdown.js

```diff
@@ -53,6 +53,10 @@ const gtEntity = /&gt;/gm;
 const ampEntity = /&amp;/gm;
 
 class Renderer extends marked.Renderer {
+  constructor() {
+    super();
```

Comment:
Why do we need this? Looks like a complete redundancy.

Reply:
Not sure either, but lint started shouting (see previous commit)

Reply:
Wait. We do need to extend this using `super()`, right?

Reply:
That's weird. Calling `renderer: new Renderer(undefined)` works :thinking:

Reply:
```
> npm run lint

> respec@25.8.0 lint C:\Users\Kagami\Documents\GitHub\respec
> tsc -p src/jsconfig.json && eslint .

src/core/markdown.js:90:15 - error TS2554: Expected 1 arguments, but got 0.

90     renderer: new Renderer(),
                 ~~~~~~~~~~~~~~

  node_modules/marked/src/Renderer.js:11:15
    11   constructor(options) {
                     ~~~~~~~
    An argument for 'options' was not provided.


Found 1 error.
```

TypeScript is getting an incorrect inferred type. Could you try installing `@types/marked`?

Reply:
Samesies. Just installed :smile: